RT_UPDATE/download_SRB.R

```diff
@@ -0,0 +1,177 @@
+#' Download or Process Sex Ratio at Birth (SRB)
+#'
+#' Retrieves country-specific Sex Ratio at Birth (SRB) data from the most recent installed \code{wpp} package
+#' (e.g., \code{wpp2022}) or returns user-supplied SRB values for a specified set of years. If the requested
+#' location is not found in WPP data, a default SRB value (1.0486) is used. Missing years are filled with the
+#' mean of available SRB values.
+#'
+#' @param SRB Optional numeric value or vector specifying SRB(s) to use. If provided, this value is repeated for all \code{DatesOut}.
+#' @param location Character or numeric code specifying the country or region for which SRB data should be downloaded.
+#' @param DatesOut Numeric vector of years for which SRB values are required. Must contain at least one element.
+#' @param verbose Logical; if \code{TRUE}, prints progress and diagnostic messages. Default is \code{TRUE}.
+#'
+#' @details
+#' The function identifies the latest installed \code{wpp} package (e.g., \code{wpp2022}) and extracts single-year
+#' SRB data (\code{sexRatio1}). It supports both country names and ISO numeric codes as \code{location} inputs.
+#' 
+#' If the specified location is not found in the WPP data, a default SRB value corresponding to a sex ratio of
+#' 105 male births per 100 female births (\code{1.0486}) is returned. If certain requested years are missing from
+#' the WPP dataset, the missing SRB values are imputed using the mean of available SRB values.
+#'
+#' @return
+#' A named numeric vector of SRB values for each year in \code{DatesOut}.
+#'
+#' @importFrom dplyr filter select
+#' @importFrom tidyr pivot_longer
+#' @importFrom tibble as_tibble
+#' @importFrom rlang .data
+#' @importFrom magrittr %>% 
+#'
+#' @examples
+#' \dontrun{
+#' # Example 1: Download SRB data for France
+#' srb_data <- download_SRB(location = "France",
+#'                          DatesOut = c(2000, 2005, 2010))
+#'
+#' # Example 2: Use user-supplied constant SRB value
+#' srb_data <- download_SRB(SRB = 1.05,
+#'                          location = "France",
+#'                          DatesOut = c(2000, 2005, 2010))
+#' }
+#'
+#' @export
+
+download_SRB <- function(SRB = NULL, 
+                         location, 
+                         DatesOut, 
+                         verbose = TRUE) {
+  
+  SRB_default <- round((1 - 0.4886) / 0.4886, 3)
```

Comment:
if you use this value, always add #KW value in a comment. Otherwise people will ask why you didn't use 1.05